sgt-back-end/index.js

```diff
@@ -0,0 +1,133 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+const JSONParser = express.json();
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.use(JSONParser);
+
+app.get('/api/grades', (req, res) => {
+  const grades = `
+    select *
+      from "grades"
+  `;
+
+  db.query(grades)
+    .then(result => {
+      res.status(200).json(result.rows);
+    })
+    .catch(err => {
+      if (err) {
+        // eslint-disable-next-line no-console
+        console.log(err);
+        res.status(500).json({ error: 'An unexpected error occurred.' });
+      }
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const input = req.body;
+  if (!input.name || !input.course || !input.score) {
+    res.status(400).json({ error: 'Name, Course and Score are required fields!' });
+    return;
+  }
+  if (isNaN(+input.score) || +input.score < 0 || +input.score > 100) {
+    res.status(400).json({ error: 'Score should be a number between 0 and 100!' });
+    return;
+  }
+  const addGrade = `
+    insert into "grades" ("name", "course", "score")
+    values ($1, $, $3)
+    returning *
+  `;
+
+  const values = [input.name, input.course, JSON.parse(input.score, 10)];
```

Comment:
You mean `parseInt`. `JSON.parse` only takes one argument.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse